packages/crowdstrike/manifest.yml

```
@@ -1,6 +1,6 @@
 name: crowdstrike
 title: CrowdStrike
-version: "1.42.2"
+version: "1.42.3"
```

```suggestion
version: "1.42.2"
```

No need for updating version. Its tests-only change.